index.html

```diff
@@ -1051,16 +1051,22 @@ <h3>Data delivery</h3>
               and {{PressureRecord/[[Time]]}} set to |timestamp|.
             </li>
             <li>
-              If |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] exists,
-              set observer.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] to |record|
-              and [=iteration/continue=].
+              Increase |observer|.{{PressureObserver/[[ChangesCountMap]]}}[|source|].
```

Comment:
The call to https://www.w3.org/TR/compute-pressure/#dfn-passes-rate-obfuscation-test below also increments [[ChangesCountMap]], so you're effectively increasing it twice, aren't you?

Reply:
Yes, I agree, there is a mistake here. It shouldn't be increased here.